main.go

```diff
@@ -129,10 +114,9 @@ func loadConfig() *Config {
 	config := new(Config)
 
 	defaultSslInsecure := envConfigBoolWithDefault("SSL_INSECURE", true)
-	defaultCertPath := envConfigStringWithDefault("PROXY_CERT_PATH", "/proxy/certs") 
+	defaultCertPath := envConfigStringWithDefault("PROXY_CERT_PATH", "/Users/lkolluru/working-dir/apple/go-gcsproxy/test/") 
```

Comment:
it's better to have a common path for default. i.e. /usr/local/gcs-proxy.  Probably don't want to use your personal dir "Users/lkolluru".